GiTinder/Controllers/UsersController.cs

```diff
@@ -45,5 +47,35 @@ public async Task<GeneralApiResponseBody> Login([FromBody] LoginRequestBody logi
             }
             return responseBody;
         }
```

Comment:
@Jonathan-Bonnin but if the /available endpoint is here, you should move the /login endpoint into a different controller

Reply:
@adamgyulavari controllers split in the next commit :)